doc/news/_preparation_next_release.md

```diff
@@ -188,6 +188,12 @@ Thanks to Daniel Bugl.
   commonly used specification keywords to be used with the [typechecker](https://www.libelektra.org/plugins/typechecker). Currently the
   keywords `check/range`, `check/enum` and `check/validation` are supported. *(Armin Wurzinger)*
 
+### Typechecker
+
```

Comment:
Shortly describe what the plugin does, not everyone knows it.

Reply:
done